MIGRATION.md

```diff
@@ -15,11 +18,70 @@ The catalog has its own version number, as we may not require to update it with
 
 Version `1.0` of the catalog adds an important property to models `backend`, defining the type of framework required by the model to run (E.g. LLamaCPP, WhisperCPP).
 
-### Changelog
+### 🛠️ How to migrate
+
+You can either delete any existing `user-catalog` by deleting the `~/.local/share/containers/podman-desktop/extensions-storage/redhat.ai-lab/user-catalog.json`.
+
+> [!WARNING]  
+> This will remove the models you have imported from the catalog. You will be able to import it again afterward.
+
+If you want to keep the data, you can migrate it by updating certain properties.
+
+### Recipes
+
+The recipe object has a new property `backend` which define which framework is required. 
+Value accepted can be `llama-cpp`, `whisper-cpp`, `none`.
+
+Moreover, the `models` property has been changed to `recommended`.
+
+**Example**
+
+```diff
+{
+  "version": "1.0",
+  "recipes": [{
+    "id": "chatbot",
+    "description" : "This is a Streamlit chat demo application.",
+    "name" : "ChatBot",
+    "repository": "https://github.com/containers/ai-lab-recipes",
+-   "models": [
++   "recommended": [
+      "hf.instructlab.granite-7b-lab-GGUF",
+       "hf.instructlab.merlinite-7b-lab-GGUF"
+    ]
++   "backend": "llama-cpp"
+  }],
+  "models": [],
+  "categories": []
+}
+```
+
+### Models
+
+The model object has also the new property `backend`, which define which framework is required.
+We increase the security by adding a new optional `sha256` property.
```

Comment:
Would be good to have a code snippet explaining how to compute the value from the file using sha256sum

Reply:
:+1: